dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit unpacker cmake-multilib
+
+MY_P="deviceatlas-enterprise-c-${PV}"
+
+DESCRIPTION="API to detect devices based on the User-Agent HTTP header"
+HOMEPAGE="https://deviceatlas.com"
+SRC_URI="https://deviceatlas.com/download/api/enterprise/c -> ${MY_P}.zip"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~x86"
+IUSE="doc examples"
+
+RESTRICT="fetch"
```

Comment:
Please put this below dependencies. While specific order of variables is not strictly requires, it helps other people read your ebuilds and reduces the risk that someone will miss something.